site/templates/application/application.jet

```diff
@@ -80,15 +80,16 @@
         window.Shift72CheckoutOptions = {
           baseUrl: window.location.origin,
           {{if useLocalDevCheckout}}checkoutDevServerUrl: '{{checkoutDevServer}}',{{end}}
-          options: {
           links: {
-              library: `${window.location.origin}{{routeToPath("/library.html")}}`,
-              termsAndConditions: `${window.location.origin}{{routeToPath("/page/terms-and-conditions/")}}`,
-              deviceCompatibility: `${window.location.origin}{{routeToPath("/page/help/")}}`,
-              help: `${window.location.origin}{{routeToPath("/page/help/")}}`,
-              signIn: `${window.location.origin}{{routeToPath("/signin.html")}}`,
-              signUp: `${window.location.origin}{{routeToPath("/signup.html")}}`,
-            }
+            library: `${window.location.origin}{{routeToPath("/library.html")}}`,
+            termsAndConditions: `${window.location.origin}{{routeToPath("/page/terms-and-conditions/")}}`,
+            deviceCompatibility: `${window.location.origin}{{routeToPath("/page/help/")}}`,
+            help: `${window.location.origin}{{routeToPath("/page/help/")}}`,
+            signIn: `${window.location.origin}{{routeToPath("/signin.html")}}`,
+            signUp: `${window.location.origin}{{routeToPath("/signup.html")}}`,
+          },
+          getLang: () => {
+            return window.localStorage.getItem('selectedLanguage');
```

Comment:
I think you want to fallback onto the default language here:

```
let selectedLanguage = window.localStorage.getItem('selectedLanguage');

return selectedLanguage || '{{defaultLanguage}}';
```

Reply:
Is there a reason it can't be `{{lang.Code}}` ?

Reply:
Actually nevermind that... the safer option would be to get it directly from kibble:
```
getLang: () => {
  return '{{lang.Code}}';
}
```

Reply:
Oh cool, I didn't know this existed.

Reply:
Lolwut... I totally just ignored Sam's comment and wrote out a more long-winded version of it!